bodhi/server/util.py

```diff
@@ -601,7 +601,10 @@ def cmd(cmd, cwd=None):
     if out:
         log.debug(out)
```

Comment:
@bowlofeggs no worries! I think I understood and implemented your changes. I pushed my changes, too, so you can find them here. To recap:

- if an err is present, but the exit code is 0, log to DEBUG (**not** log to info like we did before)
- test to make sure stdout is logged at debug
- change commit message to reflect err output to debug (**not** log to info like we did before)
- no reason to mock the logger's info level when we're mocking the debug level